Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+/**
+ * Factored out common process for running test suite in python container
+ *
+ * @param pyimage Custom python image
+ */
+def test_with_container(String pyimage) {
+    withDockerServer([uri: PCIC_DOCKER]) {
+        def pytainer = docker.image(pyimage)
+
+        pytainer.inside {
+            // Installs
+            sh 'pip install -r requirements.txt -r test_requirements.txt -i http://tools.pacificclimate.org/pypiserver/ --trusted-host tools.pacificclimate.org'
```

Comment:
1) Use https and you won't need the `--trusted-host` argument.
2) Cleaner to set the `PIP_INDEX_URL` environment variable than to use `-i`.

Reply:
Always happy to make it cleaner!